gradescopeCronJob/gradescope_to_spreadsheet.py

```diff
@@ -110,14 +112,25 @@ def get_sub_sheet_titles_to_ids(sheet_api_instance):
     if subsheet_titles_to_ids:
         return subsheet_titles_to_ids
     print("Retrieving subsheet titles to ids")
-    sheets = sheet_api_instance.get(spreadsheetId=SPREADSHEET_ID, fields='sheets/properties').execute()
+    request = sheet_api_instance.get(spreadsheetId=SPREADSHEET_ID, fields='sheets/properties')
+    sheets = request.execute()
     subsheet_titles_to_ids = {sheet['properties']['title']: sheet['properties']['sheetId'] for sheet in
                                sheets['sheets']}
     return subsheet_titles_to_ids
 
+def backoff_handler():
+    pass
```

Comment:
suggestion: use a logging library to log each retry attempt.
Example: 
1. have a parameter `details`
2. logging.info(f"Backing off {details['wait']:0.1f} seconds after {details['tries']} tries. Exception: {details['exception']}")

Reply:
Or only log if all retries are exhausted.

Reply:
Good idea! Will log the number of attempts for every assignment to avoid crowding the output.